libjsonutils/cmake/JSONUtilsConfig.cmake.in

```diff
@@ -1,18 +1,18 @@
-get_filename_component(JSONUtils_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-include(CMakeFindDependencyMacro)
-
-list(APPEND CMAKE_MODULE_PATH ${JSONUtils_CMAKE_DIR})
-
-# NOTE Had to use find_package because find_dependency does not support COMPONENTS or MODULE until 3.8.0
-
-#find_dependency(Boost 1.55 REQUIRED COMPONENTS regex)
-#find_dependency(RapidJSON 1.0 REQUIRED MODULE)
-find_package(Boost 1.55 REQUIRED COMPONENTS regex)
-find_package(RapidJSON 1.0 REQUIRED MODULE)
+get_filename_component(JSONUTILS_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+#
+# NOTE: We had to use find_package because find_dependency does not support
```

Comment:
see comment below, maybe we should bump the version to 3.15 and use `find_dependency` here correctly, what do you say?